content/guides/concepts/dnslink.md

```diff
@@ -9,7 +9,7 @@ menu:
 
 ## What is DNSLink?
 
-DNSLink uses [DNS TXT](https://en.wikipedia.org/wiki/TXT_record) records to map a domain name (like `ipfs.io`) to an IPFS address. Because you can edit your DNS records, you can use them to always point to the latest version of an object in IPFS (remember that an IPFS object’s address changes if you modify the object). Because DNSLink uses DNS records, the names it produces are also usually easy to type and read.
+DNSLink uses [DNS TXT](https://en.wikipedia.org/wiki/TXT_record) records to map a domain name (like `ipfs.io`) to an IPFS address. Because you can edit your DNS records, you can use them to always point to the latest version of an object in IPFS (remember that an IPFS object’s address changes if you modify the object). Since DNSLink uses DNS records, you can assign names/paths/(sub)domains/whatever that are easy to type, read, and remember.
```

Comment:
> names/paths/(sub)domains/whatever

Slashes are being overused (again, originally by me); also, "whatever" sounds a little too informal.

*PS: I am new to collaboration on GitHub and I'm just starting to get the hang of it. Next time, I'll try to write serious "edit suggestions" in stead of "general ideas that need to be reworded".*

Reply:
@dreamski21 IMO it doesn’t seem too informal at all; more like friendly. Let’s leave it in! You’ve done an awesome job on these. In fact, we’d love any suggestions on any other docs you work through along your IPFS journey. This help is greatly appreciated!

Reply:
Happy to help. That's the least I could do, especially after the mess I made at [ipfs/go-ipfs](https://github.com/ipfs/go-ipfs/pulls/dreamski21) (trying to "make up for my sins" XD).